### DISCUSSION TITLE
Installation of Positron IDE (Linux Manjaro)

### DISCUSSION THREAD

**Discussion by DenisVDavydov** [posit-dev/positron#4871] [Q&A] [answered]
Installation of Positron IDE (Linux Manjaro):

Conflicting files:

positron-ide-devel-bin: /usr/share/pixmaps/com.visualstudio.code.oss.png already exists

Solution:
1) pamac build positron-ide-devel-bin
2) sudo mv /usr/share/pixmaps/com.visualstudio.code.oss.png /usr/share/pixmaps/com.visualstudio.code.oss.png.bak
3) pamac build positron-ide-devel-bin

**Request:**
1) Availability of the IDE logo in SVG format for public access.

**Comment by timtmok** [comment, marked as answer]
Would this logo work for your needs?
https://github.com/posit-dev/positron/blob/1a48e235450abf6104736a7bca7f99017d778712/src/vs/workbench/browser/media/positron-icon.svg

The issue with the conflict sounds like a real problem. I assume you already have a VS Code installation?

**Reply by DenisVDavydov** [reply]
Thank you for providing the link to the logo!
Yes, I have VS Code installed.

**Comment by timtmok** [comment]
https://github.com/posit-dev/positron/issues/4875

I created an issue for this. That file should have a different name to avoid the conflict.
